plutus-core-spec/figures/TypeSynthesis-Algorithmic.tex

```diff
@@ -24,9 +24,9 @@
     \end{prooftree}
 
     \begin{prooftree}
-        \AxiomC{$cn$ has constant signature $\constsig{tcn}$ in Figure \ref{fig:constants}}
```

Comment:
I don't think the previous version of this actually made sense.  Below the line it should probably have said `(con cn):(con tn)` or something.